Automattic/simplenote-electron
# Fix Prettier Errors

### Fix
Fixes prettier errors that were created from updating prettier.  These changes are formatting only.

### Test
1. `npm test`
2. `npm run dev`
3. Click around app
4. Check console for errors

### Review
Only one developer is required to review these changes, but anyone can perform the review.
